.github/workflows/build.yaml

```
@@ -0,0 +1,181 @@
+            echo "folder_exists=true" >> "$GITHUB_OUTPUT"
+          else
+            echo "folder_exists=false" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Build and Push Database Docker Image
+        if: ${{ steps.check-db-folder.outputs.folder_exists == 'true' }}
+        id: build_push_db_docker
+        uses: docker/build-push-action@v5
+        with:
+          platforms: linux/amd64,linux/arm64
+          context: studio-services/${{ matrix.service }}/scripts/migration 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)